app/views/steps/check_answers/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
```

Comment:
According to the design, this should be one-third

Reply:
are you sure? this is what it looks like with one-third
<img width="1035" alt="image" src="https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/26544538/366ea933-fe08-4c43-b109-1cc106c55159">

Reply:
![Screenshot 2023-07-19 at 09 51 54](https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/90189839/aa880189-a0df-4d57-9832-06b9cba118c9)
'govuk-grid-column-one-thirds'

Reply:
updated - good find, I put in one-third instead of one-thirds